src/session/__tests__/sessionReducer-test.js

```diff
@@ -38,4 +38,39 @@ describe('sessionReducer', () => {
     const newState = sessionReducer(baseState, eg.action.login_success);
     expect(newState).toEqual({ ...baseState, needsInitialFetch: true });
   });
+
+  test('DEAD_QUEUE', () => {
+    const newState = sessionReducer(baseState, eg.action.dead_queue);
+    expect(newState).toEqual({ ...baseState, needsInitialFetch: true });
+  });
+
+  test('LOGOUT', () => {
+    const newState = sessionReducer(baseState, eg.action.logout);
+    expect(newState).toEqual(baseState);
+  });
```

Comment:
For these two and also `INITIAL_FETCH_COMPLETE` below, the action contains no data at all. I think it'd be cleaner to make that transparent -- see the `CANCEL_EDIT_MESSAGE` test for a model.